include/oneapi/dpl/internal/scan_by_segment_impl.h

```diff
@@ -38,13 +38,14 @@
 #include <utility>
 #include <algorithm>
 
-#include <oneapi/dpl/pstl/algorithm_fwd.h>
-#include <oneapi/dpl/pstl/parallel_backend.h>
-#include <oneapi/dpl/pstl/hetero/utils_hetero.h>
-
-#include <oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h>
-#include <oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h>
-#include <oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h>
+#include "../pstl/glue_algorithm_ranges_impl.h"
```

Comment:
Why add `../pstl/glue_algorithm_ranges_impl.h` ?

Reply:
Oh... I guess it was added by mistake.. Thanks

Reply:
fixed